lib/apiclasses.js

```diff
@@ -35,6 +35,7 @@ function StatusMessage(refersTo, description, isError) {
 /** @const */ StatusMessage.UNSPECIFIED = 'UNSPECIFIED';
 /** @const */ StatusMessage.BREAKPOINT_SOURCE_LOCATION =
                 'BREAKPOINT_SOURCE_LOCATION';
+/** @const */ StatusMessage.BREAKPOINT_ACTION = 'BREAKPOINT_ACTION';
```

Comment:
StatusMessage.refersTo is a message schema defined by the protobuf. You cannot add a new type to enumeration. The API is going to reject this. (Is there a test?).

See documentation here of the schema here:

https://cloud.google.com/debugger/api/reference/rpc/google.devtools.clouddebugger.v2#google.devtools.clouddebugger.v2.StatusMessage